### PR TITLE
Move panels

### PR DESCRIPTION
**Pull request by jr-nicola**
- I've removed the mutations drop down from the UI since there is some scoping required as to how we actually implement it. I've left the helper functions for this in for now.
- I've also changed the UI of the panels since Erik asked that the RHS panel be collapsible instead and that the treeview widget takes up most of the screen.
- Added a CSS file to style the collapsible section

### CODE REVIEW

File: R/module_plots.R [hunk 1]
```diff
@@ -7,21 +7,23 @@ plotsUI = function(id) {
...
+                                  shiny::uiOutput(ns("download_plot_button"))
+                    ),
+                    # display plot
+                    shiny::column(9,
+                                  align = "center",
+                                  display_panel(shiny::uiOutput(ns("display_plot")))
                     )
                   )
+
   )
```
Comment (russHyde) [inline R/module_plots.R:27]: You have three (almost) identical copies of this code now. Could you extract a function that creates the UI, and accepts the table title, ID for the choice dropdown, ID for the download button, and the display panel as arguments.
```r
downloader_tab_panel <- function(title, chooser_id, download_button_id, panel) {
    shiny::tabPanel(
       title,
                  shiny::br(),
                  shiny::fluidRow(
                    # drop down menu to select dataset
                    shiny::column(3,
                                  align = "center",
                                  shiny::uiOutput(chooser_id),
                                  shiny::br(),
                                  shiny::uiOutput(download_button_id)
                    ),
                    # display data
                    shiny::column(9, align = "center", panel)
                  )
}
```

File: R/app_ui.R [hunk 2]
```diff
@@ -23,14 +23,40 @@ app_ui = function(request) {
...
+
+                                             # RDS tab
+                                             rdsUI("rds1")
+
+                          )
+                        )
+                        )
+          ), # end fluid row
+
+        # Bottom row - show tree (static html output from tfpscanner)
```
Comment (russHyde) [inline R/app_ui.R:52]: This is good. Could you explain where the expandability of the "Download Files" panel is encoded, please.

File: R/module_plots.R (hunk 1 above)
Comment (jr-nicola) [reply R/module_plots.R]: I've added this function, and updated the modules UI. Thanks - the module code looks much neater now :)

File: R/app_ui.R (hunk 2 above)
Comment (jr-nicola) [reply R/app_ui.R]: The `htmltools::tags$details` and `htmltools::tags$summary` functions create the expandable section, with just html code (no javascript required). The `htmltools::tags$summary` defines what is initially displayed as a title/preview, and the rest of the code inside `htmltools::tags$details` defines the content initially hidden.

I've added a comment into this file for future reference as well.

**Review by russHyde — APPROVED** [review]
LGTM

**Pull request merged**